### PR TITLE
chore(deps): revert update dependency react-native-shake to ^5.6.2

### PR DESCRIPTION
**Pull request by MuckT**
Reverts valora-inc/wallet#5962

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -164,7 +164,7 @@
     "react-native-restart": "^0.0.27",
     "react-native-safe-area-context": "^4.11.0",
     "react-native-screens": "^3.34.0",
-    "react-native-shake": "^5.6.2",
+    "react-native-shake": "^5.5.2",
```
Comment (satish-ravi) [inline package.json:167]:
```suggestion
    "react-native-shake": "5.5.2",
```

Comment (MuckT) [reply package.json]: Updated `package.json` and `yarn.lock` in 9b18a0e & 2baa538 respectively.

**Review by satish-ravi — APPROVED** [review]

File: yarn.lock [hunk 2]
```diff
@@ -13480,13 +12859,10 @@ react-native-screens@^3.34.0:
     react-freeze "^1.0.0"
     warn-once "^0.1.0"
 
-react-native-shake@^5.6.2:
-  version "5.6.2"
-  resolved "https://registry.yarnpkg.com/react-native-shake/-/react-native-shake-5.6.2.tgz#3d9a51c529a38dd0c1198f70c37306e1178f29d9"
-  integrity sha512-SaEwg61MFMyEpe2xMzYZnzJ/PpgDFBM3R1DvRPgiPGJAtZomH8ufUJi/bUYM+vouDu0107iPY3euTB1Go3EjOQ==
-  dependencies:
-    react "^18.2.0"
-    react-native "^0.73.6"
```
Comment (jeanregisser) [inline yarn.lock:13489]: Damn, it was pulling it as dependency instead of peerDepdenceny 🤦

**Review by jeanregisser — APPROVED** [review]
Thank you 🙏

**Pull request merged**